### PR TITLE
Output update

### PR DESCRIPTION
**Pull request by Phabbits**
Added initial content for equipment and encryption methods.
Updated output for cleaner and easier to read display.
Changed rounds to hold attack index and started Present Attack section.
Removed amount from buying encryption methods.

### CODE REVIEW

File: Test.C [hunk 1]
```diff
@@ -317,13 +321,19 @@ void equipmentStore(struct Player *playerPtr, struct Equipment equipmentStock[NU
...
+    int i;
 
+    printf("\n");
     printf("|------------------------------ Equipment  Store ------------------------------|\n");
-    for(int i = 0; i < NUM_EQUIPMENT; i++){
-        printf("%s %s %d %d\n", equipmentStock[i].name, equipmentStock[i].description, equipmentStock[i].cost, equipmentStock[i].messageAmount);
+
+    printf(" # | Equipment Name       | Cost  | Message Amount\n");
+    printf("---|----------------------|-------|---------------\n");
+    for(i=0; i<NUM_EQUIPMENT; i++){
```
Comment (eholland2000) [inline Test.C]:
```suggestion
    for(int i=0; i<NUM_EQUIPMENT; i++){
```

File: Test.C [hunk 2]
```diff
@@ -317,13 +321,19 @@ void equipmentStore(struct Player *playerPtr, struct Equipment equipmentStock[NU
     int amountSelect;
     int transactionTotal;
     char yesNo;
+    int i;
```
Comment (eholland2000) [inline Test.C]:
```suggestion
```

File: Test.C (hunk 1 above)
Comment (eholland2000) [reply Test.C]: From what I can tell we don't use i outside of the loop scope

File: Test.C [hunk 3]
```diff
@@ -130,40 +122,45 @@ int main(){
...
+        printf("Credits: %d\n", player.currentCredits);
+        printf("\n");
+        printf("Equipment Inventory:\n");
 
         for (i=0; i<NUM_EQUIPMENT; i++){
             // details about the equipment type itself is stored in equipmentStock
             // amount of this equipment type that the player has is stored in equipmentInventory
-            printf("%s: %d\n", equipmentStock[i].name, player.equipmentInventory[i]);
+            if (player.equipmentInventory[i] > 0){
+                printf("%3d %s(s)\n", player.equipmentInventory[i], equipmentStock[i].name);
```
Comment (eholland2000) [inline Test.C]:
```suggestion
                printf(" - %3d %s(s)\n", player.equipmentInventory[i], equipmentStock[i].name);
```

Comment (eholland2000) [reply Test.C]: If we are using the bullet points for encryption it might be better to include them here too

File: Test.C [hunk 4]
```diff
@@ -130,40 +122,45 @@ int main(){
...
     player.encryptionInventory = {encryptionStock[0]}; /* TODO: RANDOMLY POPULATE ENCRYPTION (Do we need to tier encryption?) */
-    player.currentCredits = 30;
+    player.currentCredits = 1000;
 
     // output pregame information
     displayBriefingMessage(&player);
 
     while (currentRound < NUM_ROUNDS){
+        printf("\n");
         printf("|================================== Round %2d ==================================|\n", currentRound);
```
Comment (eholland2000) [inline Test.C]:
```suggestion
        printf("|================================== Round %2d ==================================|\n", currentRound + 1);
```

Comment (eholland2000) [reply Test.C]: Makes round numbering display in a way that makes sense without adjusting logic of loops

File: Test.C [hunk 5]
```diff
@@ -317,13 +321,19 @@ void equipmentStore(struct Player *playerPtr, struct Equipment equipmentStock[NU
     int amountSelect;
     int transactionTotal;
     char yesNo;
+    int i;
 
+    printf("\n");
     printf("|------------------------------ Equipment  Store ------------------------------|\n");
-    for(int i = 0; i < NUM_EQUIPMENT; i++){
-        printf("%s %s %d %d\n", equipmentStock[i].name, equipmentStock[i].description, equipmentStock[i].cost, equipmentStock[i].messageAmount);
+
```
Comment (eholland2000) [inline Test.C:328]:
```suggestion
printf("You currently have %i credits\n", playerPtr->currentCredits);

```

File: Test.C [hunk 6]
```diff
@@ -363,50 +376,60 @@ void equipmentStore(struct Player *playerPtr, struct Equipment equipmentStock[NU
 ******************************************************************************/
 void encryptionStore(struct Player *playerPtr, struct Encryption encryptionStock[NUM_ENCRYPTION]){
     int encryptSelect;
-    int amountSelect;
     int transactionTotal;
     char yesNo;
 
+    printf("\n");
     printf("|------------------------------ Encryption Store ------------------------------|\n");
```
Comment (eholland2000) [inline Test.C:385]:
```suggestion
printf("You currently have %i credits\n", playerPtr->currentCredits);

```

**Review by eholland2000 — CHANGES_REQUESTED** [review]
Looks really good! A few minor suggestions and then you can merge.

File: Test.C (hunk 1 above)
Comment (Phabbits) [reply Test.C]: Ahh, I did not know of this trick!

**Review by eholland2000 — APPROVED** [review]

**Pull request merged**